examples/bug_slow_max_speed/config.json

```diff
@@ -0,0 +1,13 @@
+{
+    "simulation_time_step": 1,
+    "infra_path": "infra.json",
+    "simulation_path": "simulation.json",
+    "succession_path": "succession.json",
+    "extra_rolling_stock_dirs": [
+        "./rolling_stocks/"
```

Comment:
The idea of extra rolling stock is to share rolling stock between examples and tests.
It should be `../rolling_stocks/`. Then you need to add a `slow_rolling_stock.json` in it and use it in your `simulation.json`